### DISCUSSION TITLE
[ FF7 ] I keep getting Unhandled Exception errors and dont know how to fix it- FF77th Heaven mod

### DISCUSSION THREAD

**Discussion by DivineKai123** [julianxhokaxhiu/FFNx#359] [Q&A] [answered]
During the cutscene in mideel with cloud and tifa I keep getting this Unhandled Exception error at the same point and time and I've tried many things to fix it like changing the FFNx driver to a better version like 1.10.0.0 or the one above that and before that and It still had the same issue then I reinstalled everything and with the stock driver that 7th Heaven came with I was able to get further in the cutscene until the error message popped up again, if anyone has had this problem or can help me soon Id appreciate it
![unhandled exception](https://user-images.githubusercontent.com/98239932/150658077-205db12f-e56d-481e-9cb4-800121b0e5ba.PNG)

**Comment by julianxhokaxhiu** [comment, marked as answer]
No 7th support will be provided here. Please use the official 7th support channels. Cheers.
